book/modules/appendix1-exercises.tex

```diff
@@ -52,59 +94,253 @@
 				If so, how many? Justify your answer.
 		\end{enumerate}
 
-		\prob On Kokoro's farm, there is a cage with $35$ animals, some of which 
-		are chickens
-		and some of which are rabbits. Kokoro counted the total number of legs in the
-		cage and found that there were $94$ legs in all (notably, each chicken
-		has exactly two legs and each rabbit has four legs). Kokoro decides
-		to use this information to figure out how many chickens and how many
-		rabbits there are\footnote{ This problem based on a
-		classical Chinese problem from the ancient Chinese treatise
-		\emph{Mathematical Classic of Master Sun} (or \emph{Sunzi Suanjing}) written
-		during 3rd to 5th centuries \textsc{a.d.}}.
+		\begin{solution}
+			\begin{enumerate}
+				\item If $\vec b=\mat{7 \\ 14}$, then the vector equation
+					becomes
+					\[
+						x\mat{2 \\ 4}+y\mat{8 \\ 16}=\mat{7 \\ 14}.
+					\]
+					Convert it to a system of linear equations:
+					\[
+						\systeme{2x+8y=7,4x+16y=14}.
+					\]
+					Row reduce this system, and we get
+					\[
+						\systeme{x+4y=3.5, 0x+0y=0}.
+					\]
+					The solution to this system is then
+					\[
+						\left\{
+						\begin{array}
+							{ccc} x & = & 3.5-4t \\ y & = & t
+						\end{array}\right. (t\in \mathbb{R}).
+					\]
+					This system is consistent.
+
+				\item There are vectors $\vec{b}$ that makes the system consistent.
+					For instance, $\vec b=\mat{5 \\ 10}$ is such a vector. In
+					general, any vector $\vec b$ with $\vec b=\mat{t \\ 2t}$ for
+					some $t\in\mathbb{R}$ makes the system consistent. Since
+					there are infinitely many real numbers, we conclude that there
+					are infinitely many vectors $\vec b$ that makes the system consistent.
+
+				\item If $\vec b=\mat{5 \\ 14}$, then the vector equation
+					becomes
+					\[
+						x\mat{2 \\ 4}+y\mat{8 \\ 16}=\mat{5 \\ 12}.
+					\]
+					Convert it to a system of linear equations:
+					\[
+						\systeme{2x+8y=5,4x+16y=12}.
+					\]
+					Row reduce this system, and we get
+					\[
+						\systeme{x+4y=2.5, 0x+0y=2}.
+					\]
+					This system is inconsistent.
+
+				\item There are vectors $\vec b$ that makes the system inconsistent.
+					For instance, $\mat{10 \\ 24}$ is is such a vector. In
+					general, any vector $\vec b$ with $\vec b=\mat{5t \\ 12t}$ for
```

Comment:
This isn't general. This is particular, since you haven't specified all vectors where it's inconsistent.